lib/g_searcher_web/api_router.ex

```diff
@@ -31,6 +30,6 @@ defmodule GSearcherWeb.APIRouter do
   scope "/api", GSearcherWeb.API, as: :api do
```

Comment:
Is it possible to split the scope into two? so one can ensure the JSON API spec while the other can be any types 🤔
```elixir
scope "/api", GSearcherWeb.API, as: :api do
  pipe_through [:api, :authentication]
  plug JSONAPI.EnsureSpec

  # ...
end

scope "/api", GSearcherWeb.API, as: :api do
  pipe_through [:api, :authentication]

  post "/reports", ReportController, :create
end
```